### PR TITLE
DHFPROD-7277: Compare URIs with Each Other or Compare with the Entire…

### PR DESCRIPTION
**Pull request by AkshayBajajML**
Included the change for DHFPROD-7278: View Message/Validation when All Data option exceeds Sample Size Limit

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Comment by AkshayBajajML** [comment]
@jbelonoj reviewed and approved UI changes

File: marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.module.scss [hunk 1]
```diff
@@ -157,27 +157,42 @@
 .testMatch {
     padding-left: 44px;
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.module.scss:158]: Might want to adjust the padding left to 34px. It looks a bit inconsistent as compared to other sections in match step detail page. See the screenshot below.
image

**Review by xnikhil08 — CHANGES_REQUESTED** [review]
The background blue color spreads improperly and the name spills over to the next URI row when the URI is long.
see the screencast for details.
https://drive.google.com/file/d/1ayBOrq57KjYm8G0hixctlsIYVgJmZng6/view

File: marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.module.scss (hunk 1 above)
Comment (AkshayBajajML) [reply marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.module.scss]: Fixed both issues and is ready for review

**Review by xnikhil08 — CHANGES_REQUESTED** [review]
Looks fine in chrome but Firefox seems to be messed up a bit. See the screenshot below.
image

**Comment by AkshayBajajML** [comment]
> Looks fine in chrome but Firefox seems to be messed up a bit. See the screenshot below.
> image

Fixed!!

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**